linkerd/proxy/client-policy/src/lib.rs

```diff
@@ -206,7 +206,7 @@ impl ClientPolicy {
                 opaque: opaq::Opaque {
                     // TODO(eliza): eventually, can we configure the opaque
                     // policy to fail conns?
-                    policy: None,
+                    routes: std::sync::Arc::new([]),
```

Comment:
We can remove this TODO now, right? The empty policy will fail connections, right?

We probably ought to use a static NO_OPAQ_BACKENDS to be consistent (prevents needless heap allocations).